.azure-pipelines/apiview-sync-staging.yml

```diff
@@ -0,0 +1,31 @@
+parameters:
+- name: 'containers'
+  type: object
+  default:
+  - originals
+  - codefiles
+
+
+stages:
+  - stage: 'Main'
+    jobs:
+      - job: 'BlobCopy'
+        pool:
+          vmImage: 'vs2017-win2016'
+
+        steps:
+        - pwsh: |
+            Invoke-WebRequest -MaximumRetryCount 10 -Uri "https://aka.ms/downloadazcopy-v10-windows" -OutFile "azcopy.zip" | Wait-Process;
```

Comment:
I don't see azcopy locally in the repo anymore, so I don't have an issue with this.

Reply:
Is there a reason we have to use azcopy? Can we not use https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/deploy/azure-file-copy?view=azure-devops?

Reply:
@weshaggard : Sorry for the delayed response here. Reason for using azcopy is to use "sync" operation supported by azcopy  which attempts to copy only what is missing in destination and runs whole task faster. I didn't see that option using azure file copy task. Daily sync job doesn't have to go over al blobs in source this way.